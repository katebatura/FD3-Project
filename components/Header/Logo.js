"use strict";

import React from 'react';
import PropTypes from 'prop-types';
import { NavLink } from 'react-router-dom';
class Logo extends React.Component {

  
    render() {
  
      return (          
        <NavLink to="/" exact className="header__logo">
          <span className = "header__logo-text">VSE</span>
          <span className = "header__logo-span">MASLA</span>
        </NavLink>                        
      );
  
    }
  
  }
  
  export default Logo;