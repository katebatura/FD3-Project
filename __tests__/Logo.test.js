"use strict";

import React from 'react';
import renderer from 'react-test-renderer';

import Logo from '../components/Header/Logo';

test('внешний вид Logo', () => {

  // создаём тестовую версию компонента
  const component = renderer.create(
  <Logo />
  );

  // получаем снэпшот (HTML-снимок) компонента для сверки, что вёрстка не испортилась
  let componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();

    
});
