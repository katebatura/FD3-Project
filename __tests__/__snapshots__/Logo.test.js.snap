// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`внешний вид Logo 1`] = `
<a
  aria-current={null}
  className="header__logo"
  href="/"
  onClick={[Function]}
  style={undefined}
>
  <span
    className="header__logo-text"
  >
    VSE
  </span>
  <span
    className="header__logo-span"
  >
    MASLA
  </span>
</a>
`;
